balrog.gemspec

```diff
@@ -24,6 +24,7 @@ Gem::Specification.new do |spec|
   spec.require_paths = ["lib"]
 
   spec.add_dependency "bcrypt", "~> 3.0"
+  spec.add_dependency "rails"
```

Comment:
The README says we support >= 5. However I know we're running it on Rails 4.2. I suggest we relax what the README says, and make this `>= 4.2`, as that's the earliest we've tested it against.